docs/reference_guide.md

```diff
@@ -2939,6 +2939,12 @@ Use format specifier "%s" when printing the return value. Note that `strftime`
 does not actually return a string in bpf (kernel), the formatting happens in
 userspace.
 
+bpftrace also supports the following format string extensions:
+
+Specifier | Description
+---- | -----------
+`%f` | Microsecond as a decimal number, zero-padded on the left
+
 Examples:
```

Comment:
Maybe it is nice if an example use `%f`? (i.e., `# bpftrace -e 'i:s:1 { printf("%s\n", strftime("%H:%M:%S.%f", nsecs)); }'`)